### PR TITLE
Draft 1 of prelaunch README changes

### PR DESCRIPTION
**Pull request by colinmegill**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,20 +1,13 @@
-[![Travis Status][trav_img]][trav_site]
-
 Victory
 =======
-Victory is an opinionated set of data visualization tools built in React. This repo aggregates all of the stable Victory components so they can be conveniently included. 
-
-IMPORTANT
```
Comment (boygirl) [inline README.md]: Please leave in some kind of warning about the state of the project. Maybe call it an alpha instead of a pre-release, but with the same caveats about a somewhat unstable api

File: README.md [hunk 2]
```diff
@@ -1,20 +1,13 @@
-[![Travis Status][trav_img]][trav_site]
```
Comment (boygirl) [inline README.md]: Why is this removed?

**Comment by coopy** [comment]
@boygirl ready for re-review

**Comment by coopy** [comment]
Fixes https://github.com/FormidableLabs/victory/issues/47

**Comment by boygirl** [comment]
@colinmegill @coopy approved even though CI is failing. (known issue with `babel-eslint`)
